app/prepare.py

```diff
@@ -136,8 +137,20 @@ def update_image(method, image, file):
 
 def prepare(args: TrainArgs):
     free()
+    #init_data()
```

Comment:
doing it like this doesn't empty original vars, it creates NEW vars in local context with empty value while old one still exist in global context and are never cleared.

workaround would be to explicitly state you're not defining new by saying `globals passed`, but its better to 
instead of `passed = []` do `passed.clear()` so it will use existing var declaration and clear its content.

btw, in case of bucketized, it should be `bucketized['ar'].clear(), ...`